### PR TITLE
Update Puma dependency to build on Ruby 2.5

### PR DESCRIPTION
**Pull request by ryanwi**
Puma 2 fails to install on more recent environments including Ruby 2.5. It will also fail on Codeship's new Ubuntu bionic images.

https://github.com/puma/puma/issues/1380

### CODE REVIEW

**Comment by ryanwi** [comment]
```
NoMethodError: undefined method `options' for #<Puma::CLI:0x0000561d316fc850>
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/command_receivers/http_server.rb:33:in `initialize_puma'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/command_receivers/http_server.rb:12:in `initialize'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/runner.rb:159:in `new'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/runner.rb:159:in `setup_receivers'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/runner.rb:67:in `start'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/cli.rb:37:in `execute'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/lib/emque/consuming/cli.rb:24:in `initialize'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/bin/emque:5:in `new'
  /dozer_bundle/bundler/gems/emque-consuming-b80c8aebbcbb/bin/emque:5:in `<top (required)>'
  /dozer_bundle/bin/emque:23:in `load'
  /dozer_bundle/bin/emque:23:in `<top (required)>'
```

File: lib/emque/consuming/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Emque
   module Consuming
-    VERSION = "1.4.0"
+    VERSION = "1.5.0"
```
Comment (semmons99) [inline lib/emque/consuming/version.rb]: @ryanwi given the problems with puma, should we be releasing this as a major version? I think the right answer is, “no” but thought it was worth a discussion.

Comment (ryanwi) [reply lib/emque/consuming/version.rb]: I was leaning towards the minor bump since puma is only used for the control/status.

Comment (semmons99) [reply lib/emque/consuming/version.rb]: good call

**Pull request merged**